R/casechange-functions.R

```diff
@@ -17,7 +17,14 @@ tokens_tolower <- function(x, keep_acronyms = FALSE, ...) {
 #' @noRd
 #' @export
 tokens_tolower.tokens <- function(x, keep_acronyms = FALSE, ...) {
-    types(x) <- char_tolower(types(x), keep_acronyms = keep_acronyms, ...)
+    type <- types(x)
+    if (keep_acronyms) {
+        is_acronyms <- stri_detect_regex(type, "^\\p{Uppercase_Letter}{2,}$", ...)
```

Comment:
Could be refined for the condition `\\p{Uppercase_Letter}(\\p{Uppercase_Letter}|\\d)+` to also get `B2B`, `EC2`, etc.